### PR TITLE
Show company 'complete' status to admin, members

### PR DESCRIPTION
**Pull request by weedySeaDragon**
## Changes proposed in this pull request:
1.  for admin only: display the "complete" status in the list of companies
2. for admin and company members: display the "complete" status on the company page
2. added locale entries
3.  added checkbox helpers and pre-defined icons
3.  moved all company-related features to the `features/companies` directory

### sortable column in list of Companies (index view): (only admin sees this)
co-index-complete-col-sv

---
---

### Company page: company information is complete (admin and company members only)
co-show-complete-sv

---
---
### Company page: company information is _not_ complete (admin and company members only)

co-show-incomplete-en

co-show-incomplete-sv

---
## Ready for review:
@thesuss @AgileVentures/shf-project-team

### CODE REVIEW

File: config/locales/sv.yml [hunk 1]
```diff
@@ -845,6 +845,10 @@ sv:
       dinkurs_fetching: Hämtar händelser ... vänligen vänta
       dinkurs_key: *dinkurs_company_key
       pay_branding_fee: &pay_branding_fee Betala H-märkespremie
+      info_is_complete: Info komplett
```
Comment (thesuss) [inline config/locales/sv.yml]: Din företagssida är synlig

File: config/locales/sv.yml [hunk 2]
```diff
@@ -845,6 +845,10 @@ sv:
...
+      info_is_not_complete: Info är inte komplett
```
Comment (thesuss) [inline config/locales/sv.yml]: Din företagssida är inte synlig

File: config/locales/sv.yml [hunk 3]
```diff
@@ -845,6 +845,10 @@ sv:
...
+      info_is_not_complete_explanation: This company will not appear on the website until all required information is complete.
```
Comment (thesuss) [inline config/locales/sv.yml]: Din företagssida är bara synlig när all markerad information är angiven.

**Review by thesuss — CHANGES_REQUESTED** [review]

**Comment by patmbolger** [comment]
I'm seeing some weird styling in the company (admin) view:

Screen Shot 2021-03-20 at 4 13 43 PM

Looking at `payor/_term_status.html.haml` I see content directly within rows.  Also in other areas, e.g. that file renders:

```
- if current_user.admin?
      .admin-only
        .row
          = render partial: 'payor/admin_note_and_edit_status_row',
           locals: { entity: entity,
           target_id: admin_edit_target_id,
           button_text: admin_edit_status_text }
```
and that partial contains:

```
.admin-only

  .row.mt-2
    .col.d-flex.pt-2
      .flex.pr-3
        = payment_notes_label_and_value(entity)

.row.mt-2
  .col.d-flex.pt-2.pr-3

....
```
So rows within a row.

AFAIK, only divs of class `col` should be immediate children of rows, and rows should only have `col` elements as immediate children.

**Comment by weedySeaDragon** [comment]
@patmbolger  - agree.  But this is something that I haven't touched in this PR.  Can you open a new chore for that please?

**Pull request merged**